sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,62 @@
+# Testing
+
+To run our tests, first do a full update of the project by running `rush update`, then rebuild either this package, with `rush build -t @azure/app-configuration` or all packages with `rush build`, then make sure to build the tests with `rush build:test`. After that, you can either run our unit tests, which won't reach to live services, and our integration tests, which will indeed require existing, reachable resources.
```

Comment:
> then make sure to build the tests with `rush build:test`

The `test`, `test:node` and `test:browser` scripts do build the test code before running the tests. Any reason we want to call out building the tests specifically here?

> After that, you can either run our unit tests, which won't reach to live services, and our integration tests, which will indeed require existing, reachable resources.

I would imagine a contributor to ever just use the `test`, `test:node` or `test:browser` scripts. Both `unit` and `integration` scripts are for CI. Due to this, I would avoid calling out "unit" and "integration" tests in our readme instructions so as to not mislead the user (because our "unit" script doesnt really run "unit" tests :) ). 

Instead, I would call out that by default, the tests are run in playback mode. If you want to target live services, set TEST_MODE to "live" and set the required env variables.

Reply:
Thank you! I'll make sure the new approach has this covered.